src/UI/Implementation/Component/Input/Field/Renderer.php

```diff
@@ -56,12 +56,12 @@ protected function renderNoneGroupInput(Component\Input\Field\Input $input) {
 
 		if ($input instanceof Component\Input\Field\Text) {
 			$input_tpl = $this->getTemplate("tpl.text.html", true, true);
+		} elseif ($input instanceof Component\Input\Field\Numeric) {
+			$input_tpl = $this->getTemplate("tpl.numeric.html", true, true);
+		} elseif ($input instanceof Component\Input\Field\Password) {
+			$input_tpl = $this->getTemplate("tpl.password.html", true, true);
```

Comment:
I agree. IMO this should tackle in a different PR. I think @xus already started since during the Select-Implementation? Is that code back in the trunk?